routes/api/v1/sessions.js

```diff
@@ -0,0 +1,20 @@
+var express = require('express');
+var router = express.Router();
+var User = require('../../../models').User;
+var pry = require('pryjs');
+
+/* POST login user */
+router.post('/', function(req, res, next){
```

Comment:
Really clean approach thanks to the model functions. I really wish I had something this clean.